rag_experiment_accelerator/doc_loader/pdfLoader.py

```diff
@@ -37,6 +40,28 @@ def preprocess_pdf_content(content: str):
     return content
 
 
+def _load_pdf_using_pypdf(file_path: str) -> List[Document]:
+    loader = PyPDFLoader(file_path=file_path)
+    return loader.load()
+
+
+def _load_pdf_using_unstructured(file_path: str) -> List[Document]:
+    elements = partition_pdf(filename=file_path, strategy="ocr_only")
+    element_meta = elements[0].to_dict()["metadata"] if elements else {}
```

Comment:
Severity: Potential Issue - Using the hardcoded strategy 'ocr_only' in the `partition_pdf` function might not be suitable for all use cases. Consider making this strategy configurable through function parameters or environment settings for better flexibility.


[Hack - peer-review-pal] We are sorry for the intrusion. Feel free to ignore/delete the comments with the tag - they are automatically generated by [our Hack project](https://github.com/microsoft/peer-review-pal).
If you find them useful, want us to delete them or have any feedback or want to contribute - reach out to some of us: juliame@microsoft.com vadimkirilin@microsoft.com oteslenko@microsoft.com vimeena@microsoft.com
You can prevent future comments from appearing by adding #prp-ignore to your PR description.